sdk/objectanchors/Azure.MixedReality.ObjectAnchors/src/Customizations/Models/AssetConversionConfiguration.cs

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Azure.Core;
+using Azure.MixedReality.ObjectAnchors.Models;
+
+namespace Azure.MixedReality.ObjectAnchors
+{
+    /// <summary> Represents an asset conversion configuration. </summary>
+    [CodeGenModel("IngestionConfiguration")]
+    public partial class AssetConversionConfiguration
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AssetConversionConfiguration"/> class.
+        /// </summary>
+        /// <param name="gravity">The asset gravity, which must be normalized.</param>
+        /// <param name="scale">The asset scale.</param>
+        internal AssetConversionConfiguration(System.Numerics.Vector3 gravity, float scale) : this(new Vector3(gravity), scale)
+        {
+            if (!gravity.IsNormalized())
+            {
+                throw new ArgumentException("The value must be normalized.", nameof(gravity));
+            }
+        }
+
+        internal AssetConversionConfiguration(Vector3 gravityWrapper, float scale)
```

Comment:
You documented the other one. Might as well do the same for this one.

Reply:
Used to be public once upon a time, documentation is just an artifact of that. Removing.